PWGGA/GammaConv/macros/AddTask_TRDEfficiency.C

```diff
@@ -38,7 +38,61 @@ AliAnalysisTRDEfficiency* AddTask_TRDEfficiency(
 
     // ================== GetInputEventHandler =============================
     AliVEventHandler *inputHandler=mgr->GetInputEventHandler();
-    
+    cout << "$$$$$$$$$ part of the task $$$$$$$$" << endl;
+    //=========  Set Cutnumber for V0Reader ================================
+    TString cutnumberPhoton = "00200009227300008250400000";
+    TString cutnumberEvent = "00010113";
```

Comment:
the event cut looks strange, the V0reader should run a preselection - this looks like a 'complete' event cut